docs-src/go/introduction-to-go-sdk.md

```diff
@@ -0,0 +1,129 @@
+---
+id: introduction-to-go-sdk
+title: Introduction to the Temporal Go SDK developer's guide
+description: The Temporal Go SDK provides the benefits of both the Temporal programming model and the ease of writing in Go.
+sidebar_label: Go SDK
+tags:
+  - guide-context
+---
+
+Welcome to the Temporal Go SDK developer’s guide!
+
+:::success Temporal Go SDK API reference
+
+https://pkg.go.dev/go.temporal.io/sdk
+
+:::
+
+Temporal Python SDK first released on September 29, 2020, and has been enabling Python developers to build Temporal Applications ever since.
```

Comment:
> Temporal Python SDK first released on September 29, 2020, and has been enabling Python developers to build Temporal Applications ever since.

I call shenanigans on this claim. 

According to the [releases page in GitHub](https://github.com/temporalio/sdk-python/releases), the Python SDK was first released (as alpha 0.1a1) on March 18, 2022. In fact, [the first commit](https://github.com/temporalio/sdk-python/commit/75f7113428f64c8a2c3f3f540f005331a2d0fa37) in the repo was only about two months earlier, or well over a year after the September 29, 2020 initial release date claimed here.

Reply:
Of course, the bigger issue is that this is talking about the release of the Python SDK in a document intended for Go developers. Although I would argue that initial release date of the Python SDK isn't even relevant to a Python developer, it's certainly not relevant to a Go developer.